### DISCUSSION TITLE
ModuleNotFoundError: No module named 'ultralytics'

### DISCUSSION THREAD

**Discussion by le-khang** [Bing-su/adetailer#205] [Q&A] [answered]
I got this error from several days ago. I uninstalled and reinstailled the extension but it doesn't work. Then I tried to download adetailer May version and it still doesn't work. Please help.
![2023-07-14 11_16_22-My Pods](https://github.com/Bing-su/adetailer/assets/35536989/b70e555b-bc16-4c93-8345-b9eefe9ac590)

Also I don't understand what you said by "Put your [ultralytics](https://github.com/ultralytics/ultralytics) model in webui/models/adetailer". What model should I put in that folder? Where do I download it? I git clone that ultralytics repo and found no file ends with .pt or .pth in it.

![2023-07-14 14_02_23-Bing-su_adetailer_ Auto detecting, masking and inpainting with detection model](https://github.com/Bing-su/adetailer/assets/35536989/64b0f6a4-f6b7-4f28-b251-96e4e30f8570)

**Comment by Bing-su** [comment]
Please post the full console logs as I'm asking for in the bug report. I would want to see logs of the installation phase as well.

![Cap 2023-07-14 16-12-10-540](https://github.com/Bing-su/adetailer/assets/37621276/6221f6dc-6651-46e6-a41e-e93827e6f250)

**Comment by le-khang** [comment]
Thank you for your quick response. Unfortunately I'm using Runpod and they didn't show any logs of the installation phase. I uninstalled and reinstalled but the log shows nothing at all. Below is the full logs when I tried to create 1 image:

![2023-07-14 15_45_58-My Pods](https://github.com/Bing-su/adetailer/assets/35536989/44927922-724c-4127-a7a2-ab302089f49c)
![2023-07-14 15_47_54-My Pods](https://github.com/Bing-su/adetailer/assets/35536989/6e753614-595c-42e9-abd5-92d960a46bce)
![2023-07-14 15_48_40-My Pods](https://github.com/Bing-su/adetailer/assets/35536989/9d5e4618-2098-4d6c-bf1f-b8d9db444741)
![2023-07-14 15_49_23-My Pods](https://github.com/Bing-su/adetailer/assets/35536989/2e330baf-e948-4954-8e3c-cf7807483877)
![2023-07-14 15_49_36-My Pods](https://github.com/Bing-su/adetailer/assets/35536989/ec5768ce-0577-4283-9cd4-2f41b94555a0)
![2023-07-14 15_49_45-My Pods](https://github.com/Bing-su/adetailer/assets/35536989/51c73bb0-7dcd-4867-a52a-18cb4188667a)

Also I checked the folder /workspace/stable-diffusion-webui/models/adetailer and there is nothing in it. Is that alright?

![2023-07-14 15_44_22-JupyterLab](https://github.com/Bing-su/adetailer/assets/35536989/793e7972-b531-4f79-b15b-8110faeec103)
![2023-07-14 15_43_58-JupyterLab](https://github.com/Bing-su/adetailer/assets/35536989/b20ad1e3-b949-4657-8954-b455e179e406)

**Comment by Bing-su** [comment, marked as answer]
It's normal for the models folder to be empty.

![콘솔로그좀 다 올려라 인간들아](https://github.com/Bing-su/adetailer/assets/37621276/af642790-3319-43a8-9bdf-25f1b841cac6)
You set the skip-install flag, so it won't be installed, and when I say post the full console log, I mean post ALL 344 lines of your console.

**Reply by le-khang** [reply]
Sorry I didn't know that. I thought there are too many so I tried to choose the most relevant ones that I could think of. And thank you very much, it works now.

**Comment by RedOceanCC** [comment]
> Sorry I didn't know that. I thought there are too many so I tried to choose the most relevant ones that I could think of. And thank you very much, it works now.

Hi, how did you solve this problem? What do you need to do with --skip-install ?

**Reply by phopeu** [reply]
Deleting "--skip-install" from one of the webui files fixed it for me. I think it was the webui-user.sh 
Then full restart of the server is needed for the change to be reflected.

**Reply by YangYT1234567** [reply]
Hello 
I encountered the same problem,How did you solve it?"--skip-install" Where exactly is it located and how to delete it? webui-user.sh  Do files need to be deleted?Please help share the solution, thank you very much！
